api.js

```diff
@@ -77,8 +77,10 @@ module.exports = function attachAPI(app, {io, db}) {
 
     // Extra details for a channel - these aren't returned in the channel list API,
     // but are when a specific channel is fetched.
-    channelDetail: c => Object.assign(serialize.channelShort(c), {
-      pinnedMessageIDs: c.pinnedMessageIDs
+    channelDetail: async c => Object.assign(serialize.channelShort(c), {
```

Comment:
> async

Ouch!

Would be nice for _all_ `serialize` methods to be async for consistency.

Reply:
Agreed.

Reply:
Fixed!